gramps/gen/lib/grampstype.py

```diff
@@ -165,41 +179,27 @@ def __set_tuple(self, value):
         self.__value = val
         self.__string = strg
 
-    def __set_int(self, value):
+    @set.register
+    def set_int(self, value: int):
```

Comment:
This should be private.

Reply:
This has been addressed in the latest commit